src/commands/dao_cmds/token.js

```diff
@@ -0,0 +1,131 @@
+const TaskList = require('listr')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const { getContract } = require('../../util')
+const listrOpts = require('../../helpers/listr-options')
+
+const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
+
+exports.command =
+  'token <token-name> <symbol> [decimal-units] [transfer-enabled]'
```

Comment:
On second thought, should we also put this under `aragon` instead of `dao` ?
Doing help on this command shows:
```
Positionals:
  dao         Address of the Kernel or AragonID)                        [string]
  token-name  Full name of the new Token                              [required]
  symbol      Symbol of the new Token                                 [required]
```
even though `dao token name symbol` works somehow :face_with_head_bandage:

Reply:
I guess `dao token new` it is :D